src/main/java/picard/sam/SetNmMdAndUqTags.java

```diff
@@ -102,10 +105,16 @@ protected int doWork() {
 
         final ReferenceSequenceFileWalker refSeq = new ReferenceSequenceFileWalker(REFERENCE_SEQUENCE);
 
-        StreamSupport.stream(reader.spliterator(),false)
-                .peek(rec->{if(!rec.getReadUnmappedFlag()) AbstractAlignmentMerger.fixNmMdAndUq(rec, refSeq, IS_BISULFITE_SEQUENCE);})
+        StreamSupport.stream(reader.spliterator(), false)
+                .peek(rec -> {
+                    if (!rec.getReadUnmappedFlag()) {
+                        if (SET_ONLY_UQ)
```

Comment:
This is now complicated enough that I would advocate moving back to non-lambda format. also needs `{` and `}` in if/else